test/e2e/data/infrastructure-azure/v1beta1/cluster-template.yaml

```diff
@@ -29,24 +29,16 @@ spec:
   namespace: tigera-operator
   releaseName: projectcalico
   repoURL: https://docs.tigera.io/calico/charts
-  valuesTemplate: |-
-    installation:
-      cni:
-        type: Calico
-      calicoNetwork:
-        bgp: Disabled
-        mtu: 1350
-        ipPools:
-        ipPools:{{range $i, $cidr := .Cluster.spec.clusterNetwork.pods.cidrBlocks }}
-        - cidr: {{ $cidr }}
-          encapsulation: VXLAN{{end}}
-      registry: mcr.microsoft.com/oss
-    # Image and registry configuration for the tigera/operator pod.
-    tigeraOperator:
-      image: tigera/operator
-      registry: mcr.microsoft.com/oss
-    calicoctl:
-      image: mcr.microsoft.com/oss/calico/ctl
+  valuesTemplate: "installation:\n  cni:\n    type: Calico\n  calicoNetwork:\n    bgp:
+    Disabled\n    windowsDataplane: HNS\n    mtu: 1350\n    ipPools:\n    ipPools:{{range
+    $i, $cidr := .Cluster.spec.clusterNetwork.pods.cidrBlocks }}\n    - cidr: {{ $cidr
+    }}\n      encapsulation: VXLAN{{end}}\n  serviceCIDRs: \n    - 10.96.0.0/12 #
+    must match cluster service CIDR (this is the default)\n# when kubernetesServiceEndpoint
+    (required for windows) is added \n# DNS configuration is needed to look up the
+    api server name properly\n# https://github.com/projectcalico/calico/issues/9536\ndnsConfig:
+    \n  nameservers:\n    - 127.0.0.53\n  options:\n    - name: edns0\n    - name:
+    trust-ad\nkubernetesServiceEndpoint:\n  host: \"{{ .Cluster.spec.controlPlaneEndpoint.host
+    }}\"\n  port: \"{{ .Cluster.spec.controlPlaneEndpoint.port }}\""
```

Comment:
Can this be formatted like the previous diff so it's a bit more readable rather than using newline characters?

Reply:
I can take a look, I think this is an automatically generated file?

Reply:
got it, it was a ` ` (space) 🤦